doc-build/setup.py

```diff
@@ -15,7 +15,7 @@ def get_version():
         return f.read().strip()
 
 setup(
-    name='doc_build',
+    name='docer',
```

Comment:
Should the package name be the same as folder name?

Reply:
it may confuse others as `doc_build` is similar to HF's `doc_builder`.